tests/test_PostEndpoint.py

```diff
@@ -0,0 +1,26 @@
+
+
+
+from c20_server.PostEndpoint import app
+
+
+def empty_json():
```

Comment:
I believe test functions must be named with strings starting with `test_`.  So this test would be `test_empty_json`.

Do these tests run when you execute `pytest`?